qemu/qemu.go

```diff
@@ -2280,6 +2313,9 @@ type Knobs struct {
 
 	// IOMMUPlatform will enable IOMMU for supported devices
 	IOMMUPlatform bool
+
+	// MemEncrypt will enable memory encryption
+	MemEncrypt bool
```

Comment:
TODO: change to ConfidentialGuest bool if it still required

Reply:
That sounds like a good idea.

Reply:
I think this flag is unnecessary as the same bit of information can be assumed when the client sets `ObjectType = "sev-guest"`

Reply:
Uh, maybe.  It might be convenient to have this flag somewhere global-ish, rather than having to scan the object list for it, but whatever works for you, really.